frontend/src/common/constansts/validationErrorMessages.ts

```diff
@@ -1,5 +1,6 @@
 export enum ValidationErrorMessages {
     EMAIL_NOT_VALID = "Please enter a valid email address",
     NAME_NOT_VALID = "Name must contain at least 4 symbols",
-    PASSWORD_NOT_VALID = "Password must contain at least 4 symbols "
+    PASSWORD_NOT_VALID = "Password must contain at least 4 symbols ",
```

Comment:
```suggestion
    PASSWORD_NOT_VALID = "Password must contain at least TEXT_FIELD_VALIDATION symbols ",
```

TEXT_FIELD_VALIDATION -> PASSWORD_MIN_SIZE